lib/index.js

```diff
@@ -64,7 +67,7 @@ function getUserFavorites(api_key){
                                   dataType: 'json',
                                   success: function(res){
                                     userFavoritesObj = new UserFavorites(res["data"]);
-                                    displayFavorites();
+                                    callback();
                                   },
                                   error: function(res){
```

Comment:
Super minor thing here. Instead of declaring res as the response variable in the error block, it is common to use `e` or `error`. It is more stylistic than anything else, but it can help with your code readability. You may also want to have the error be something that is user friendly so that you can show what error was hit. Something like -
```
error: function(e) {
  userFavoritesObj = "Error: " + e;
}
```